### ISSUE TITLE
Radar chart's labels don't show on desktop

### ISSUE THREAD

**Issue by synw** [intkhabahmed/multi_charts#6]
Everything works well using the radar chart in a desktop app except the labels that do not show. No text is visible. It works on mobile fine with the same code.

For info I compiled the desktop app using [Go Flutter](https://github.com/go-flutter-desktop/go-flutter)

**Comment by intkhabahmed** [comment]
Did you try increasing the textScaleFactor value? Try increasing it, if you haven't and let me know.

**Comment by synw** [comment]
I tried with 2.0 text scale, nothing is shown. Does it use a special font? If the font is not installed it does not render on desktop: this happens with go-flutter when the Roboto font is not installed, so maybe this can be a similar issue?

**Comment by intkhabahmed** [comment]
No, I haven't used any explicit font-family. I have used Canvas to draw text. I don't know what's the actual issue, maybe it is the issue from the Go Flutter which you mentioned

**Comment by synw** [comment]
Maybe. This problem remains mysterious. Try it with go-flutter if you can to reproduce (and btw if you do not know go-flutter give it a try, it works well).

**Comment by intkhabahmed** [comment]
Sure I will try go-flutter, and will try to find the cause
